vscode-dotnet-runtime-extension/package.json

```diff
@@ -66,7 +67,11 @@
 				},
 				"dotnetAcquisitionExtension.existingDotnetPath": {
 					"type": "array",
-					"description": "File Path to an existing installation of .NET."
+					"description": "File Path to an existing installation of a .NET Runtime."
+				},
+				"dotnetAcquisitionExtension.existingDotnetSDKPath": {
+					"type": "array",
+					"description": "File Path to an existing installation of a .NET SDK Folder."
```

Comment:
Big question - these are going to effectively be the same thing for a lot of users. Meaning I think lots of folks are setting these to the existing global install locations. Do we need both settings? Can we mark one deprecated and point to the more general one? How are these used distinctly in the code?

Reply:
Hm. When developing the SDK extension I didn't know what the app host was so I made it return the folder such as C:\Program Files\dotnet\sdk\7.0.113 because I thought there'd be a .NET exe in there, obviously there is not.

Some of the validation of 'did it install the right version of the sdk' is, checking for that versioned folder to exist.
We can return the same folder as the dotnet runtime such as C:\Program Files\dotnet, (this currently returns the same thing but with sdk at the end) though that doesnt guarantee the apphost will use the older sdk that someone might've requested if there's a new sdk on the machine (in fact, it wont, though not like adding the sdk folder to the path changes that.)

Someone could have used an old path to an app host that doesn't have any .NET SDK installs... I guess that'd still be fine and hopefully the app host will find it... given this information, do you still think its ok for them to share the existing variable? If so, I see no need to deprecate it.

Reply:
TLDR there is some mismatch between using the apphost folder as output for installing a specific sdk version, but I think this is the best choice so Ill make that change now and remove this variable

Reply:
I made them the same which required some changes